pennylane/operation.py

```diff
@@ -2404,7 +2404,7 @@ def check_wires_partial_overlap(self):
         return 0
 
     def sparse_matrix(
-        self, wires=None, format="csr"
+        self, wires=None, wire_order=None, format="csr"
```

Comment:
now that I look at it, since `wire_order` is the preferred one, I'd put it first.
```suggestion
        self, wire_order=None, wires=None, format="csr"
```